.appveyor/appveyor_helper.bat

```diff
@@ -0,0 +1,40 @@
+@echo off
+setlocal enableDelayedExpansion
```

Comment:
Nice question 👍 
I used enableDelayedExpansion for following two reason:
1. To expand variables at execution time rather than parse time (see [here](https://ss64.com/nt/delayedexpansion.html) for more information
2. Restore original environment at end of batch file (see [here](https://stackoverflow.com/a/6680005) or ```setlocal /?``` in cmd for more information)